docs/source/cpp/compute.rst

```diff
@@ -1367,6 +1367,10 @@ value, but smaller than nulls.
 +-----------------------+------------+-----------------------------+-------------------+--------------------------------+----------------+
 | sort_indices          | Unary      | Boolean, Numeric, Temporal  | UInt64            | :struct:`SortOptions`          | \(2) \(5)      |
 +-----------------------+------------+-----------------------------+-------------------+--------------------------------+----------------+
+| select_k              | Unary      | Binary- and String-like     | UInt64            | :struct:`SelectKOptions`       | \(5) \(3) \(6) |
++-----------------------+------------+-----------------------------+-------------------+--------------------------------+----------------+
+| select_k              | Unary      | Boolean, Numeric, Temporal  | UInt64            | :struct:`SelectKOptions`       | \(5) \(6)      |
++-----------------------+------------+-----------------------------+-------------------+--------------------------------+----------------+
```

Comment:
Why is this row duplicated twice? We can keep all the notes together (it should be clear that (3) only applies to binary/string types only…)

Reply:
Also, please try to keep the table sorted alphabetically.